web/src/tippyjs.ts

```diff
@@ -111,6 +111,14 @@ export function initialize(): void {
         placement: "bottom",
     });
 
+    delegate("body", {
+        target: ".edit-attachment-buttons",
+        trigger: "mouseenter",
+        delay: EXTRA_LONG_HOVER_DELAY,
+        appendTo: () => document.body,
+        placement: "bottom",
+    });
```

Comment:
This is unlikely to be the solution we want for the misalignment of the tooltips for the Uploaded files > attachment buttons.

Reply:
As you can see from the screenshot, the tool tip is still not aligned with the icons. ![image](https://github.com/zulip/zulip/assets/82862779/369ec08c-fad6-462f-b10d-93e8884f464e)

Reply:
We could fix this issue by shifting the padding away from the tooltip reference element. But on the other hand we could use this opportunity to instead replace these with the squared icons we use in every other place in the settings menu. (Check image below)

![image](https://github.com/zulip/zulip/assets/82862779/e9051b90-e654-4d2f-bd81-18ac6204f39a)

Reply:
Let's do this in a separate commit, after removing the previous changes on the `.edit-attachment-buttons`. Also, while we are at it, let's use the sea green color we use over at "Preview profile" for the "Download" button.

![image](https://github.com/zulip/zulip/assets/82862779/481eacb9-003b-426c-89c6-6ed590565a45)

@alya does this sound like a good idea to you?